build/azure-pipelines/build-product.yml

```diff
@@ -51,31 +51,25 @@ stages:
       pool:
         vmImage: $(imageName)
       steps:
-        - task: PowerShell@2
-          displayName: Set last tag to variable
-          name: getVersion
-          inputs:
-            targetType: 'inline'
-            script: |
-              $VERSION_TAG = git describe --tags (git rev-list --tags --max-count=1)
-              Write-Host("##vso[task.setvariable variable=VERSION_TAG]$VERSION_TAG")
-              Write-Host($VERSION_TAG)
+        - template: build-tag.yml
+        - script: |
+            echo $(getVersion.VERSION_TAG)
         - template: build-common.yml
           parameters:
             OS: $(os)
             Arch: $(arch)
             ArtifactName: $(artifact)
-            VersionTag: $(VERSION_TAG)
+            VersionTag: $(getVersion.VERSION_TAG)
 
   - stage: CreatePackages
     displayName: Create packages to publish
     jobs:
     - job: Sign_and_pack
       pool:
         vmImage: 'windows-latest'
-      variables:
-        versionTag: $[ stageDependencies.Compile.Sqlcmd.outputs['getVersion.VERSION_TAG'] ]
       steps:
+      - template: build-tag.yml
+      - template: build-common.yml

```

Comment:
we don't need to build again, we are downloading the build from the previous stage in the next step

Reply:
Good catch! Had added it mistakenly. Removed in last commit.